api/src/main/java/org/openmrs/Person.java

```diff
@@ -39,7 +39,7 @@
  * 
  * @see org.openmrs.Patient
  */
-public class Person extends BaseChangeableOpenmrsData {
+public class Person extends BaseCustomizableData PersonAttribute implements Customizable PersonAttribute {
```

Comment:
BaseChangeableOpenmrsData is a core domain object for openmrs projects, So changing it may not extend other attributes of person class.

Reply:
Did you see travis failure